js/main.js

```diff
@@ -25,8 +25,9 @@
                 id = currentPhoto.id;
                 title = currentPhoto.title;
                 secret = currentPhoto.secret;
-                picUrl = "https://farm"+farm+".staticflickr.com/"+server+"/"+id+"_"+secret+"_z.jpg";
-                $('.image-slider-wrapper ul').append("<li class='active imgArray' ><a data-lightbox='image-set'" + " href='" + picUrl + "'><img class='active' "+ " src='" +picUrl+"'/></a></li>");
+                picUrl = "https://farm" + farm + ".staticflickr.com/" + server + "/" + id + "_" + secret + "_z.jpg";
+
+                $('#images').append("<div class='col-xs-6 col-md-3 col-lg-1'><a data-lightbox='image-set'" + " href='" + picUrl + "'><img class='img-responsive' "+ " src='" +picUrl+"'/></a></div>");
```

Comment:
The `<div>` here will need the custom class name added. `.col-xs-6, .col-md-3, .col-lg-1 .custom-class`